package.json

```diff
@@ -3999,6 +3999,446 @@
 											}
 										]
 									},
+									{
+										"key": "languageId:julia",
+										"name": "Julia",
+										"type": "bindings",
+										"bindings": [
+											{
+												"key": "x",
```

Comment:
`x` usually has the meaning of delete, so it is a little bit weird. This is common action, maybe we can delete this and relied on the binding of `SPC m s s`.

Reply:
It's super common, so I changed it to `,`. That's nicer anyway (`,` - `,`).